client/guidestours/index.js

```diff
@@ -1,125 +1,20 @@
 /**
  * External dependencies
  */
-import React, { Component, PropTypes } from 'react'
+import React, { Component } from 'react'
```

Comment:
Interesting. `client/layout` is a neat place for it. I've created an issue for this, and naming. Created #4575